eng/source-build-patches/0001-Conditionally-remove-net472-from-TargetFrameworks.patch

```diff
@@ -0,0 +1,32 @@
+From b3ea2bb3b55f654b32475f214e3d3a4b39e3d3eb Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Mon, 15 Jul 2019 19:26:42 +0000
+Subject: [PATCH 1/5] Conditionally remove net472 from TargetFrameworks
+
+---
+ .../Microsoft.Net.Compilers.Toolset.Package.csproj             | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj b/src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj
+index 9e646ebab5b..7a27d77877f 100644
+--- a/src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj
++++ b/src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj
+@@ -2,6 +2,7 @@
+ <Project Sdk="Microsoft.NET.Sdk">
+   <PropertyGroup>
+     <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
++    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">netcoreapp3.1</TargetFrameworks>
```

Comment:
I don't really understand the point of these tiny patches vs just updating the source right now. It would be simpler to review, have less chance of messing up particular legs, and have less maintenance burden. I'd much rather we just make the change in one step.